### ISSUE TITLE
Calculate ss on real data

### ISSUE THREAD

**Issue by cdquinto** [agladstein/SimPrily#73]
Hi, we want to use SimPrily for GBS data (without the ascertainment bias correction) and without a recombination map (since there's no recombination map the species of interest). 

The script **real_data_ss.py** expects array data:
` parser.add_argument("-a","--array", help="The location of the array file",required=True)`

but it seems that it checks the param and model file to change the value of using_pseudo_array:

`using_pseudo_array = True
    if not processedData.get('discovery') and not processedData.get('sample') and not processedData.get('daf'):
        using_pseudo_array = False`

Is there a way to walk around this (besides giving a bogus file as array)?

Since we don't have a recombination map, we don't use the -R option in the model file and the simulations seem to be working well. But somehow this is expected when running  **real_data_ss.py**. (see error)

Thanks! 
![WhatsApp Image 2019-04-12 at 7 20 46 PM](https://user-images.githubusercontent.com/13246531/56158298-49822380-5f87-11e9-87dc-f00247df125c.jpeg)
[param_tlacua_4pops.txt](https://github.com/agladstein/SimPrily/files/3081696/param_tlacua_4pops.txt)

[model_tlacua_4pops.txt](https://github.com/agladstein/SimPrily/files/3081699/model_tlacua_4pops.txt)

**Comment by cdquinto** [comment]
we had to comment line 703 in process_input_files to get a result

**Comment by agladstein** [comment]
For now you can just give the same genome file as the array file.

```
python real_data_ss.py -m model.csv -p param.txt -o out_real -g genome_data -a genome_data
```
And you do not need to include the `-discovery`, `-sample`, `-daf`, `-random_discovery` arguments in the model.csv file.

I also fixed the genetic_map error 82d6b30100d7fd000a2c1aa092cfe67b153a5359
